package.json

```diff
@@ -2,6 +2,11 @@
   "scripts": {
     "test": "xo"
   },
+  "dependencies": {
+    "grunt": "^1.0.1",
+    "grunt-cli": "^1.2.0",
+    "grunt-webstore-upload": "^0.8.10"
+  },
```

Comment:
Another option (I'm not keen on) is to install all the grunt stuff in the `before_install` and then not include it here at all. No one needs it for development.